pep-0570.rst

```diff
@@ -59,14 +60,28 @@ builtin functions that can specify positional-only
 parameters and pure Python implementations that lack the
 syntax for it. The '/' syntax is already exposed in the
 documentation of some builtins and interfaces generated by
-the argument clinic. Making positional-only arguments a possibility
-in Python will make the language more consistent and make it clearer
-to users that positional-only arguments are allowed in builtins and argument
-clinic C interfaces.
-
-We can find positional-only arguments useful in several situations. Example:
-we want to create a function that converts from one type to
-another::
+the argument clinic. 
+
+Making positional-only arguments a possibility in Python will make the
+language more consistent and since it would be a normal feature of Python
+rather than a feature exclusive to extension modules, it should reduce
+surprise and confusion by users encountering functions with positional-only
+arguments. Notably, major third-party packages are already using the "/"
+notation in their interfaces [#numpy-ufuncs]_ [#scipy-gammaln]_.
+
+Positional-only arguments may be useful in several situations. One of the more
+extreme situations is in a function that can take any keyword parameter but
+also can take a positional one. Well-known examples for this situation are
+``Formatter.format`` and ``dict.update``. For instance, ``dict.update``
+accepts a dictionary (positionally) and/or any set of keyword parameters to use
+as key/value pairs. In this case, if the dictionary parameter were not
```

Comment:
Note: We need to change the implementation to make this work, and it might make things quite more complex.

Reply:
What do you mean? This is working with the current implementation

Reply:
Hummmmm, I see. Indeed, not sure if this is even possible. Maybe we should modify this paragraph....